Add Xtremepush (Actions) Destination docs. (#6528)
segmentio/segment-docs: src/connections/destinations/catalog/actions-xtremepush/index.md

---
title: Xtremepush (Actions) Destination

> ---

Author: @forstisabella Thanks for your suggestions and I'll apply all of them soon. But a note for this one is that we want to make our new Xtremepush (Actions) Destination public beta after merging this doc, so I will pass `hidden: false` instead.

{% include components/actions-fields.html %}

## Identify

If you're not familiar with the Segment Specs, take a look to understand what the [Identify method](/docs/connections/spec/identify/) does. An example call would look like:

```
analytics.identify('userId123', {
  email: 'john.doe@example.com',
  phone: '1234567890',
  firstName: 'John'
});
```

When you identify a user, we'll pass that user's information to Xtremepush and will try to update or create a new user based on whether a Profile exists with that `user_id`.

Some special traits will also be used as additional user identifiers:

| Segment Trait | Xtremepush User Identifier |
| ------------- | -------------------------- |
| email         | email                      |
| phone         | mobile_number              |

For any additional traits you want to save you should create [User Profile Attributes](https://docs.xtremepush.com/docs/attributes-tags){:target="_blank"} in your Xtremepush Project.

> If a trait does not match a custom Xtremepush User Profile Attribute and is not recognized as a User Identifier it will be ignored.

Reviewer: ```suggestion
If a trait does not match a custom Xtremepush User Profile Attribute and is not recognized as a User Identifier, Xtremepush ignores the trait.
```

## Track

If you're not familiar with the Segment Specs, take a look to understand what the [Track method](/docs/connections/spec/track/) does. An example call would look like:

```
analytics.track('Product Purchased', {
  productName: 'Some Product'
})
```

> Track calls will be sent to Xtremepush as a `event hits`, so you can use it to [trigger a campaign](https://docs.xtremepush.com/docs/campaign-events){:target="_blank"} for a user.

Reviewer: ```suggestion
Track calls are sent to Xtremepush as a `event hits` and you can use them to [trigger a campaign](https://docs.xtremepush.com/docs/campaign-events){:target="_blank"} for a user.
```

Event properties can be used as merge tags in the message content. You can also define additional rules on where to trigger the campaign based on event properties value.

## Enabling Push and In-App Notifications

> To enable Xtremepush push and in-app notifications you will also need to to install the relevant Xtremepush SDKs.

Reviewer: ```suggestion
To enable Xtremepush push and in-app notifications you must also install the relevant Xtremepush SDKs.
```

---
title: Xtremepush (Actions) Destination
beta: true

> hidden: false

Reviewer: ```suggestion
```

Reviewer: we can remove this attribute if hidden is not set to true, as it's not hidden by default!
